libyul/backends/wasm/WasmCodeTransform.cpp

```diff
@@ -90,7 +90,7 @@ wasm::Expression WasmCodeTransform::generateMultiAssignment(
 	return { std::move(block) };
 }
 
-wasm::Expression WasmCodeTransform::operator()(VariableDeclaration const& _varDecl)
+wasm::Expression WasmCodeTransform::operator()(yul::VariableDeclaration const& _varDecl)
```

Comment:
Why add all these?

Reply:
Because half of the code uses that, where there is a conflict between Yul AST name and Wasm AST name. I find it confusing that half is prefixed, the other half is not.

Reply:
It was confusing for me too. Would have been best if we used prefixes for both.